README.md

```diff
@@ -145,6 +145,8 @@ later steps are identical, so those steps have been covered in one place.
 
 #### Windows and Linux
 
+1. Install python-pip
+    1. `sudo apt install python-pip`
```

Comment:
See the "Linux" section earlier in this document. There we install pip via `python ./get-pip.py --user` (using the previously downloaded script from upstream) and then setup a virtualenv. If you believe that installing via package is better, it may be worth moving this step specifically to the "Linux" section and then updating the rest of the doc to replace the earlier references to installing via the `get-pip.py` script. I don't recall how many there are, there might just be the one.